<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Title</title>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style>
      code {
        background-color: red;
      }
    </style>
  </head>
  <body>
    <article>
      <p>
        All principles are visualized with RNA and DNA as example analytes.
        Principle 1 is based on physical separation of the distinct molecular
        analytes (parts a,b). a, Following complete lysis of the isolated cell
        or nucleus, poly(A) RNA hybridizes to oligo-dT-coated paramagnetic
        beads, and following magnetic pulldown, the supernatant that contains
        the genomic DNA is transferred to a new reaction vessel7,9.
        Alternatively, biotinylated nucleotides are incorporated into
        RNA-derived cDNA, allowing their capture with streptavidin-coated
        paramagnetic beads (not shown)42. Advantages include flexibility in
        downstream processing of DNA and/or RNA and compatibility with intact
        cells and nuclei from fresh and frozen tissue. Disadvantages include
        potential loss of RNA and/or DNA molecules during physical separation.
        b, In an alternative approach, lysis conditions that rupture the cell
        but not the nuclear envelope allow separation of nuclear from
        cytoplasmic molecular analytes, either by precipitating the nucleus with
        centrifugation or magnetic pulldown followed by aspiration of the
        cytosolic supernatant13,14 or by microfluidic-controlled
        nucleus-from-cytoplasm separation16,17. Advantages include flexibility
        in downstream processing of DNA and RNA and availability of non-poly(A)
        RNA. Disadvantages include loss of nuclear RNA plus some cytoplasmic RNA
        during nuclear–cytoplasmic separation, loss of mitochondrial analytes,
        need for intact single cells, incompatibility with frozen tissue and
        likely incompatibility with mitotic cells (in which the nuclear envelope
        disaggregates). c, In principle 2, termed preamplification and split,
        distinct analytes are differentially tagged and jointly preamplified,
        followed by splitting the preamplification reaction for analyte-specific
        sequencing library preparations18. Advantages include minimal risk of
        analyte loss and compatibility with intact cells and nuclei from fresh
        and frozen tissue. Disadvantages include limited flexibility, as the
        preamplification protocol needs to be suitable for all analytes of
        interest, and risk of cross-contaminating molecular analytes. d,
        Principle 3, termed seq-split, involves analyte-specific barcoding and
        sequencing library preparation in a single-pot reaction20. Multi-omic
        information is uncoupled computationally following sequencing.
        Advantages include minimal risk of analyte loss and compatibility with
        intact cells and nuclei from fresh and frozen tissue. Disadvantages
        include that libraries cannot be sequenced separately to optimal depth
        for each modality, and potential risk of cross-contaminating molecular
        analytes. e, In principle 4, termed combinatorial indexing, molecular
        analytes of single cells are tagged without isolating single cells21.
        Multiple cells or nuclei are deposited per well of a multi-well plate,
        whereby each cell or nucleus serves as a reaction container. Each
        receives an analyte-specific tag and a well-specific barcode. By
        pooling, mixing and randomly re-distributing the cells or nuclei in
        subsequent rounds of well-specific barcoding, molecular analytes
        uniquely barcoded per cell are obtained. Combination with concepts of
        principle 2 and/or 3 achieves single-cell or single-nucleus multi-omics.
        Advantages include that there is no need for isolating single cells,
        ability to achieve extremely high throughput, and compatibility with
        intact cells and nuclei from fresh and frozen tissue. Disadvantages
        include typically lower sensitivity, risk of analyte loss and limited
        flexibility in whole-genome and whole-transcriptome amplification
        protocols. dsDNA, double-stranded DNA; NGS, next generation sequencing;
        poly(A), polyadenylated.
      </p>
    </article>
  </body>
</html>
